schemas/v3.1/schema.yaml

```diff
@@ -1,7 +1,7 @@
-$id: 'https://spec.openapis.org/oas/3.1/schema/2022-10-07'
+$id: 'https://spec.openapis.org/oas/3.1/schema/WORK-IN-PROGRESS'
 $schema: 'https://json-schema.org/draft/2020-12/schema'
 
-description: The description of OpenAPI v3.1.x documents without schema validation, as defined by https://spec.openapis.org/oas/v3.1.0
+description: The description of OpenAPI v3.1.x Documents without Schema Object validation
```

Comment:
"without Schema Object Validation": should we rather use `schema-base.yaml` in our schema tests?

Reply:
Definitely, but see also #4147 - `schema-base` would work for us but it's not quite the default OAS behavior, and we should take advantage of that to provide a more intuitively-named schema that validates that behavior.  Note that `schema-base` is correct for a slightly different, more restrictive use case.